nightly.Jenkinsfile

```diff
@@ -8,7 +8,8 @@ node {
             payload = JsonOutput.toJson([
                     username: "Realm CI",
                     icon_emoji: ":jenkins:",
-                    text: "*The current realm-core nightly build was ok!*\n<${run.absoluteUrl}|Click here> to check the build."
+                    text: "@realm-core-engineers *The current realm-core nightly build was ok!*\n<${run.absoluteUrl}|Click here> to check the build."
```

Comment:
can we not @here the channel for successful builds please?

Reply:
You don't want to ... ok… so I'll close the PR..